pkg/collector/corechecks/networkpath/networkpath.go

```diff
@@ -59,20 +59,40 @@ func (c *Check) Run() error {
 		return fmt.Errorf("failed to trace path: %w", err)
 	}
 
+	// Add tags to path
+	commonTags := c.getCommonTags()
+	path.Tags = commonTags
+
 	// send to EP
 	err = c.SendNetPathMDToEP(senderInstance, path)
 	if err != nil {
 		return fmt.Errorf("failed to send network path metadata: %w", err)
 	}
 
-	tags := c.getCommonTags()
-	c.submitTelemetryMetrics(senderInstance, path, startTime, tags)
+	metricTags := c.getCommonTagsForMetrics()
+	metricTags = append(metricTags, commonTags...)
+	c.submitTelemetryMetrics(senderInstance, path, startTime, metricTags)
 
 	senderInstance.Commit()
 	return nil
 }
 
 func (c *Check) getCommonTags() []string {
+	tags := utils.CopyStrings(c.config.Tags)
+
+	agentHost, err := hostname.Get(context.TODO())
+	if err != nil {
+		log.Warnf("Error getting the hostname: %v", err)
+	} else {
+		tags = append(tags, "agent_host:"+agentHost)
```

Comment:
Are we setting this as `agent_host` instead of `host` because it's intended to go on all of the hops?

Reply:
@stb2132 
I used `agent_host` since it's commonly used to track which agent is sending some metrics/events for example in NDM SNMP Integration.

And `agent_host` is more explicit than `host` alone :)

Reply:
Yup, ok makes sense.